config_default.go

```diff
@@ -360,6 +387,13 @@ func (c *Config) GetAuthorizeCodeLifespan(_ context.Context) time.Duration {
 	return c.AuthorizeCodeLifespan
 }
 
+func (c *Config) GetDeviceAndUserCodeLifespan(_ context.Context) time.Duration {
+	if c.AuthorizeCodeLifespan == 0 {
```

Comment:
Shouldn't it be `DeviceAndUserCodeLifespan`?

Reply:
Good catch :)